packages/playground-defaults/tests/add.test.js

```diff
@@ -0,0 +1,326 @@
+const fs = require("fs")
```

Comment:
Applies to other tests as well I think. It would be nice to document in the beginning what is being tested

Reply:
done 5c99976efd1eb11e8056ff56ad8f71cb6bf83867